ast/ddl.go

```diff
@@ -1280,13 +1281,15 @@ func (n *TableOption) Restore(ctx *RestoreCtx) error {
 		}
 	case TableOptionStatsPersistent:
 		// TODO: not support
+		ctx.WriteKeyWord("STATS_PERSISTENT = DEFAULT")
 		ctx.WritePlain(" /* TableOptionStatsPersistent is not supported */ ")
```

Comment:
Do we still need this?

Reply:
yes, it's necessary.
The following two sql corresponds to two different ast tree:
```
create table t (c int) STATS_PERSISTENT = default

create table t (c int);
```
To ensure that the two ast trees are equal, we have to add `STATS_PERSISTENT = default`

Reply:
I mean is the comment `/* TableOptionStatsPersistent is not supported */` still needed to be printed out?

Reply:
yes, It's used to remind people that `STATS_PERSISTENT = default` is not the right result.